src/pretix/base/signals.py

```diff
@@ -122,3 +122,8 @@ def send(self, sender: Event, **named) -> List[Tuple[Callable, Any]]:
 idempotent, i.e. it should not make a difference if this is sent out more often
 than expected.
 """
+register_admin_settings = django.dispatch.Signal()
+"""
+All plugins that are installed may send fields for the admin settings form, as
+a list of tuples (setting name, form field).
```

Comment:
In other places where we have APIs like this (payment providers and ticket output providers), we expects dict-like collections of form fields (and all sample implementations use ``OrderedDict``). Do you think we should go for consistency here?